<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrbitMainWindow</class>
 <widget class="QMainWindow" name="OrbitMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2481</width>
    <height>1138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orbit Profiler</string>
  </property>
  <property name="accessibleName">
   <string>Orbit Profiler</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit[clearButtonEnabled=&quot;true&quot;] {
	background-image: url(:/actions/search_small_offset);
	background-position: left center;
	background-repeat: no-repeat;
	height: 22px;
	padding-left: 20px;
	padding-bottom: 1px;
}

QToolTip {
	color: #ffffff;
	background-color: #2a82da;
	border: 1px
	solid white;
}

QPushButton:disabled {
	color: gray;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="MainTabWidget">
       <property name="accessibleName">
        <string>MainTabWidget</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="CaptureTab">
        <attribute name="title">
         <string>Capture</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolBar" name="capture_toolbar">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="accessibleName">
            <string>CaptureToolBar</string>
           </property>
           <addaction name="actionToggle_Capture"/>
           <addaction name="actionCaptureOptions"/>
           <addaction name="actionHelp"/>
           <addaction name="actionIntrospection"/>
           <addaction name="actionConfigureTracks"/>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>7</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSplitter" name="captureWindowSplitter">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="opaqueResize">
               <bool>true</bool>
              </property>
              <property name="handleWidth">
               <number>20</number>
              </property>
              <property name="childrenCollapsible">
               <bool>false</bool>
              </property>
              <widget class="orbit_qt::TrackConfigurationWidget" name="trackConfig" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="accessibleName">
                <string>TrackConfigurationPane</string>
               </property>
              </widget>
              <widget class="OrbitGLWidget" name="CaptureGLWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="rightTabsAndLogSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QTabWidget" name="RightTabWidget">
        <property name="accessibleName">
         <string>RightTabWidget</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="SymbolsTab">
         <attribute name="title">
          <string>Symbols</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QSplitter" name="splitter_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="OrbitDataViewPanel" name="ModulesList" native="true">
              <property name="accessibleName">
               <string>ModulesDataView</string>
              </property>
             </widget>
             <widget class="OrbitDataViewPanel" name="PresetsList" native="true">
              <property name="accessibleName">
               <string>PresetsDataView</string>
              </property>
             </widget>
            </widget>
            <widget class="OrbitDataViewPanel" name="FunctionsList" native="true">
             <property name="accessibleName">
              <string>FunctionsDataView</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="liveTab">
         <attribute name="title">
          <string>Live</string>
         </attribute>
         <layout class="QGridLayout" name="liveGridLayout">
          <item row="0" column="0">
           <widget class="OrbitLiveFunctions" name="liveFunctions" native="true">
            <property name="accessibleName">
             <string>LiveDataView</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="samplingTab">
         <attribute name="title">
          <string>Sampling</string>
         </attribute>
         <layout class="QGridLayout" name="samplingGridLayout">
          <item row="0" column="0">
           <widget class="OrbitSamplingReport" name="samplingReport" native="true">
            <property name="accessibleName">
             <string>SamplingDataView</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="OrbitSamplingReport" name="inspectionReport" native="true">
            <property name="visible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="topDownTab">
         <attribute name="title">
          <string>Top-Down</string>
         </attribute>
         <layout class="QGridLayout" name="topDownGridLayout">
          <item row="0" column="0">
           <widget class="CallTreeWidget" name="topDownWidget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="bottomUpTab">
         <attribute name="title">
          <string>Bottom-Up</string>
         </attribute>
         <layout class="QGridLayout" name="bottomUpGridLayout">
          <item row="0" column="0">
           <widget class="CallTreeWidget" name="bottomUpWidget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="selectionSamplingTab">
         <attribute name="title">
          <string>Sampling (selection)</string>
         </attribute>
         <layout class="QGridLayout" name="selectionGridLayout">
          <item row="0" column="0">
           <widget class="OrbitSamplingReport" name="selectionReport" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="selectionTopDownTab">
         <attribute name="title">
          <string>Top-Down (selection)</string>
         </attribute>
         <layout class="QGridLayout" name="selectionTopDownGridLayout">
          <item row="0" column="0">
           <widget class="CallTreeWidget" name="selectionTopDownWidget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="selectionBottomUpTab">
         <attribute name="title">
          <string>Bottom-Up (selection)</string>
         </attribute>
         <layout class="QGridLayout" name="selectionBottomUpGridLayout">
          <item row="0" column="0">
           <widget class="CallTreeWidget" name="selectionBottomUpWidget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tracepointsTab">
         <attribute name="title">
          <string>Tracepoints</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="OrbitDataViewPanel" name="TracepointsList" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="debugTab">
         <attribute name="title">
          <string>Debug</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="orbit_qt::DebugTabWidget" name="debugTabWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="captureLogWidget">
        <property name="visible">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="captureLogVerticalLayout">
         <property name="leftMargin">
          <number>13</number>
         </property>
         <property name="topMargin">
          <number>13</number>
         </property>
         <property name="rightMargin">
          <number>13</number>
         </property>
         <property name="bottomMargin">
          <number>13</number>
         </property>
         <item>
          <widget class="QTextEdit" name="captureLogTextEdit">
           <property name="accessibleName">
            <string>CaptureLogTextEdit</string>
           </property>
           <property name="tabStopDistance">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2481</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Capture"/>
    <addaction name="actionRename_Capture_File"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Preset"/>
    <addaction name="actionSave_Preset_As"/>
    <addaction name="separator"/>
    <addaction name="actionEnd_Session"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionOpenUserDataDirectory"/>
    <addaction name="actionOpenAppDataDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFeedback">
    <property name="title">
     <string>Feedback</string>
    </property>
    <addaction name="actionReport_Missing_Feature"/>
    <addaction name="actionReport_Bug"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <widget class="QMenu" name="menuCrash">
     <property name="title">
      <string>Crash Test</string>
     </property>
     <addaction name="actionCheckFalse"/>
     <addaction name="actionStackOverflow"/>
     <addaction name="actionServiceCheckFalse"/>
     <addaction name="actionServiceStackOverflow"/>
    </widget>
    <addaction name="menuCrash"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionConfigureTracks"/>
    <addaction name="actionIntrospection"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionCaptureOptions"/>
    <addaction name="actionSourcePathMappings"/>
    <addaction name="actionSymbolLocationsDialog"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuDebug"/>
   <addaction name="menuHelp"/>
   <addaction name="menuFeedback"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReport_Missing_Feature">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/feedback</normaloff>:/actions/feedback</iconset>
   </property>
   <property name="text">
    <string>Report Missing Feature</string>
   </property>
  </action>
  <action name="actionReport_Bug">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/bug_report</normaloff>:/actions/bug_report</iconset>
   </property>
   <property name="text">
    <string>Report Bug</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/info</normaloff>:/actions/info</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_Preset">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/bookmarks</normaloff>:/actions/bookmarks</iconset>
   </property>
   <property name="text">
    <string>Open Preset...</string>
   </property>
  </action>
  <action name="actionToggle_Capture">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/play_arrow</normaloff>:/actions/play_arrow</iconset>
   </property>
   <property name="text">
    <string>Toggle Capture</string>
   </property>
   <property name="toolTip">
    <string>Toggle Capture (F5)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionCaptureOptions">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/settings</normaloff>:/actions/settings</iconset>
   </property>
   <property name="text">
    <string>Capture Options...</string>
   </property>
   <property name="toolTip">
    <string>Open the Capture Options Dialog</string>
   </property>
  </action>
  <action name="actionFilter_Tracks">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/filter_list</normaloff>:/actions/filter_list</iconset>
   </property>
   <property name="text">
    <string>Filter Tracks</string>
   </property>
   <property name="toolTip">
    <string>Filter Tracks</string>
   </property>
  </action>
  <action name="actionFilter_Functions">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/search</normaloff>:/actions/search</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="toolTip">
    <string>Search</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/help</normaloff>:/actions/help</iconset>
   </property>
   <property name="text">
    <string>Capture Window Controls</string>
   </property>
   <property name="toolTip">
    <string>Toggle display of the capture window controls (Ctrl + H)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionIntrospection">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/info</normaloff>:/actions/info</iconset>
   </property>
   <property name="text">
    <string>Introspection Window</string>
   </property>
   <property name="toolTip">
    <string>Toggle display of the introspection window</string>
   </property>
  </action>
  <action name="actionEnd_Session">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/arrow_back</normaloff>:/actions/arrow_back</iconset>
   </property>
   <property name="text">
    <string>End Session</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/exit_to_app</normaloff>:/actions/exit_to_app</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionToogleDevMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dev Mode</string>
   </property>
  </action>
  <action name="actionSave_Preset_As">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/bookmark_border</normaloff>:/actions/bookmark_border</iconset>
   </property>
   <property name="text">
    <string>Save Preset As...</string>
   </property>
  </action>
  <action name="actionOpen_Capture">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/folder</normaloff>:/actions/folder</iconset>
   </property>
   <property name="text">
    <string>Open Capture...</string>
   </property>
  </action>
  <action name="actionCheckFalse">
   <property name="text">
    <string>Check False</string>
   </property>
  </action>
  <action name="actionStackOverflow">
   <property name="text">
    <string>Stack Overflow</string>
   </property>
  </action>
  <action name="actionServiceCheckFalse">
   <property name="text">
    <string>[Service] Check False</string>
   </property>
  </action>
  <action name="actionServiceStackOverflow">
   <property name="text">
    <string>[Service] Stack Overflow</string>
   </property>
  </action>
  <action name="actionOpenUserDataDirectory">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/folder</normaloff>:/actions/folder</iconset>
   </property>
   <property name="text">
    <string>Open User Data Directory...</string>
   </property>
   <property name="toolTip">
    <string>This location contains presets and autosaved captures.</string>
   </property>
  </action>
  <action name="actionOpenAppDataDirectory">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/folder</normaloff>:/actions/folder</iconset>
   </property>
   <property name="text">
    <string>Open App Data Directory...</string>
   </property>
   <property name="toolTip">
    <string>This location contains logs, crash dumps, the symbol file cache, and the configuration file that specifies where to search for symbol files (SymbolPaths.txt).

It used to also contain captures and presets, but these were moved to the User Data directory.</string>
   </property>
  </action>
  <action name="actionConfigureTracks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/track_settings</normaloff>:/actions/track_settings</iconset>
   </property>
   <property name="text">
    <string>Track Configuration Pane</string>
   </property>
   <property name="toolTip">
    <string>Toggles the track configuration pane</string>
   </property>
  </action>
  <action name="actionSourcePathMappings">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/source_path_mappings</normaloff>:/actions/source_path_mappings</iconset>
   </property>
   <property name="text">
    <string>Source Paths Mappings...</string>
   </property>
   <property name="toolTip">
    <string>Configure path mappings for source code</string>
   </property>
  </action>
  <action name="actionSymbolLocationsDialog">
   <property name="icon">
    <iconset resource="../../icons/orbiticons.qrc">
     <normaloff>:/actions/source_path_mappings</normaloff>:/actions/source_path_mappings</iconset>
   </property>
   <property name="text">
    <string>Symbol Locations...</string>
   </property>
   <property name="toolTip">
    <string>Add or remove local symbol directories</string>
   </property>
  </action>
  <action name="actionRename_Capture_File">
   <property name="text">
    <string>Rename/Move Capture File...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OrbitDataViewPanel</class>
   <extends>QWidget</extends>
   <header>OrbitQt/orbitdataviewpanel.h</header>
  </customwidget>
  <customwidget>
   <class>OrbitGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>OrbitQt/orbitglwidget.h</header>
  </customwidget>
  <customwidget>
   <class>OrbitLiveFunctions</class>
   <extends>QWidget</extends>
   <header>OrbitQt/orbitlivefunctions.h</header>
  </customwidget>
  <customwidget>
   <class>OrbitSamplingReport</class>
   <extends>QWidget</extends>
   <header>OrbitQt/orbitsamplingreport.h</header>
  </customwidget>
  <customwidget>
   <class>CallTreeWidget</class>
   <extends>QWidget</extends>
   <header>OrbitQt/CallTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>orbit_qt::TrackConfigurationWidget</class>
   <extends>QWidget</extends>
   <header>OrbitQt/TrackConfigurationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>orbit_qt::DebugTabWidget</class>
   <extends>QWidget</extends>
   <header>OrbitQt/DebugTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/orbiticons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>OnFilterTracksTextChanged(QString)</slot>
  <slot>OnFilterFunctionsTextChanged(QString)</slot>
 </slots>
</ui>
